test/e2e/app-dir/navigation/navigation.test.ts

```diff
@@ -178,8 +178,6 @@ describe('app dir - navigation', () => {
 
       if (isNextStart || isNextDeploy) {
         await browser.waitForIdleNetwork()
-        // there should be an RSC call for the prefetch
-        expect(hasRscRequest).toBe(true)
```

Comment:
since we're now able to share the auto prefetch, the router doesn't attempt to prefetch the page with search params anymore, since it'd return the same information.